02-start.Rmd

```diff
@@ -39,28 +46,28 @@ If you wish to access the platform from another formal HMG technology solution (
 
 You must not access the platform from a non-MOJ device, such as a personal mobile phone or laptop.
 
-When you first access your platform account, you will be asked to set up additional two factor authentication for the platform itself.  You must do this using an authentication app on your personal or work phone (such as [Authy](https://authy.com/download/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en)).  There is no SMS option.
+When you first access your platform account, you will be asked to set up additional two-factor authentication for the platform itself.  You must do this using an authentication app, as no SMS option is available. For the reasons described in Section \@ref(two-factor-authentication), we recommend that you use [Authy](https://authy.com/download/).
```

Comment:
"on your personal or work phone" seems useful reassurance that people are allowed to use their personal phone, or have I missed something?

Reply:
Clarified this in 596f6a1.